src/plugins/intel_cpu/tests/unit/snippets_transformations/x64/lowered/buffer_allocation.cpp

```diff
@@ -113,13 +113,34 @@ class BufferAllocationCPUTest : public testing::TestWithParam<BufferAllocationCP
 
     virtual std::shared_ptr<ov::Model> GetModel(const std::vector<ov::PartialShape>& shapes) const = 0;
 
-    void MarkOp(const std::shared_ptr<ov::Node>& node, const std::vector<size_t>& subtensor) const {
-        for (const auto& input : node->inputs())
-            ov::snippets::lowered::PortDescriptorUtils::set_port_descriptor_ptr(
-                input, std::make_shared<ov::snippets::lowered::PortDescriptor>(input, subtensor));
-        for (const auto& output : node->outputs())
-            ov::snippets::lowered::PortDescriptorUtils::set_port_descriptor_ptr(
-                output, std::make_shared<ov::snippets::lowered::PortDescriptor>(output, subtensor));
+    void MarkOp(const std::shared_ptr<ov::Node>& node, const std::vector<std::vector<size_t>>& subtensors) const {
```

Comment:
Just an idea how to make the code more obvious: maybe we can pass vector of input subtensors (size of the vector must be equal to number of inputs)? From my perspective, it is more readable:
Was:
```
// Was
MarkOp(brgemm_cpu0, {subtensor_full, subtensor_flat}); // need to know operation semantic to understand which subtensor corresponds to the specific input

// Now
MarkOp(brgemm_cpu0, {subtensor_full, subtensor_full, subtensor_flat}); // more obvious: first 2 inputs have full subtensor, last - flat
```

I don't insist on that though

Reply:
@v-Golubev I see your point, maybe it is better to make it more explicit, but in that case we need to add another array for outputs as well (because outputs are also marked). What do you think? Is it gonna be too verbose?

Reply:
It seems like we always have only one output of ops in these tests, so I don't think `MarkOp` will be overweighted if we add a separate parameter for outputs

Reply:
Addressed